### ISSUE TITLE
Domino issue with bag slots HELP!

### ISSUE THREAD

**Issue by Rathoflite** [tullamods/Dominos#269]
I was messing around with changing my bar's and I noticed I could switch all my bags to be a 1 push button.  I could click on only one bag (showing ad gui) and they all open.  **Now I can't get it back to the original main bag plus 4 bags**.  I disabled all adons and all of the bags were showing again.  I  put all addons back one at a time.  

Finally I put Domino back in, and the issue came back, all my bags were converted to one bag again.

Also note that I am unable to make changes to that "ONE" bag.  When I pull up configuration mode  all other action bars turn blue and are capable of being edited.  The "SINGLE BAG" does says bag across the button and does react a litte (very very brief color change almost not even visible.

![wowscrnshot_020217_122205](https://cloud.githubusercontent.com/assets/25511332/22560650/63a4dae4-e943-11e6-917b-e4704a0524bd.jpg)

**Comment by Tuller** [comment]
I think this one is an addon conflict, but I don't know which addon is causing the issue.

**Comment by Rathoflite** [comment]
So if that is the case I should turn them all off except Domino, and I should be able to change it back.

**Comment by Rathoflite** [comment]
Well I removed all <span class="hiddenSpellError" pre="all ">addons except Dominos. There was no change, except that the bag icon was now useable in configuration mode.  So I just took another profile and deleted that one.  Not wasting any more time on something like this.  I am interested in how this is a conflict with another <span class="hiddenSpellError" pre="another ">addon since I disabled them all. 

Do conflicts still happen even if all were turned off?  Bliz makes it pretty easy now to turn everything off and leave whatever you want one.
